src/test/java/com/example/rets_api/converter/AnimalPolicyConverterTest.java

```diff
@@ -0,0 +1,86 @@
+package com.example.rets_api.converter;
+
+import com.example.rets_api.dto.AnimalPolicyDTO;
+import com.example.rets_api.entity.AnimalPolicyEntity;
+import com.example.rets_api.repository.UtilsTest;
+import org.junit.Test;
+
+import static com.example.rets_api.converter.PropertyConverterTest.checkAllFields_Property;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+public class AnimalPolicyConverterTest {
+
+    //DTO To Entity
+
+    @Test
+    public void dtoTOEntity_NullAnimalPolicyConverter_ShouldReturnNull(){
+        AnimalPolicyEntity animalPolicyEntity = AnimalPolicyConverter.animalPolicyDTOToAnimalPolicyEntity.convert(null);
+        assertNull(animalPolicyEntity);
+    }
+
+    @Test
+    public void dtoTOEntity_EmptyFinancialDataConverter_ShouldReturnEmpty(){
```

Comment:
Change FinancialData to AnimalPolicy at the method name

Reply:
done